integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -827,3 +828,45 @@ def render_mirrored_page_field(request, region_slug, language_slug):
             "page_form": page_form,
         },
     )
+
+
+@permission_required("cms.change_page")
+# pylint: disable=unused-argument
+def refresh_date(
+    request,
+    page_id,
+    region_slug,
+    language_slug,
+):
+    """
+    Refresh the date for all translations of a corresponding page
+
+    :param request: The current request
+    :type request: ~django.http.HttpResponse
+
+    :param page_id: The id of the page of the current page form
+    :type page_id: int
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :param language_slug: The slug of the current language
+    :type language_slug: str
+
+    :return: A redirection to the :class:`~integreat_cms.cms.views.pages.page_form_view.PageFormView`
+    :rtype: ~django.http.HttpResponseRedirect
+    """
+
+    page_translations = PageTranslation.objects.filter(page_id=page_id)
+    for page_translation in page_translations:
+        if page_translation is page_translation.latest_version:
+            page_translation.save()
+
+    # Get url of current page form for the redirect
+    language = get_object_or_404(Language, slug=language_slug)
+    page_translation = page_translations.filter(language=language).first()
+    redirect_url = page_translation.backend_edit_link
+    messages.success(
+        request, _("Successfully refreshed the date for all translations.")
+    )
+    return redirect(redirect_url)
```

Comment:
```suggestion
    return redirect(
        "edit_page",
        **{
            "page_id": page_id,
            "language_slug": language_slug,
            "region_slug": request.region.slug,
        },
    )
```
This does a few unnecessary database requests - I think this is enough.